feature/staticroute/otg_tests/static_route_test.go

```diff
@@ -0,0 +1,112 @@
+package gnmi_test
+
+import (
+	"testing"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/telemetry"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func TestGNMI(t *testing.T) {
+
+	// Configure a DUT
+
+	dut := ondatra.DUT(t, "dut")
+	dp1 := dut.Port(t, "port1")
+	dp2 := dut.Port(t, "port2")
+	// dp3 := dut.Port(t, "port3")
+	// dp4 := dut.Port(t, "port4")
+
+	ifCfg1 := &telemetry.Interface{
+		Name:        ygot.String(dp1.Name()),
+		Description: ygot.String("From Ixia"),
+	}
+
+	ifCfg1.GetOrCreateSubinterface(0).
+		GetOrCreateIpv4().
+		GetOrCreateAddress("192.0.2.10").
+		PrefixLength = ygot.Uint8(31)
+
+	ifCfg2 := &telemetry.Interface{
```

Comment:
Additionally, check out `deviations.InterfaceEnabled`. I think `Enabled` value may be required in `telemetry.Interface` and its subinterfaces.